copyparty/web/browser.js

```diff
@@ -10342,10 +10358,13 @@ function reload_browser() {
 
 	var oo = QSA('#files>tbody>tr>td:nth-child(3)');
 	for (var a = 0, aa = oo.length; a < aa; a++) {
-		var sz = oo[a].textContent.replace(/ +/g, ""),
-			hsz = sz.replace(/\B(?=(\d{3})+(?!\d))/g, " ");
-
-		oo[a].textContent = hsz;
+		var sz = oo[a].textContent.replace(/[ ,]/g, "");
+		var numSz = parseInt(sz);
+		if (!isNaN(numSz)) {
+			// Store original size as data attribute for other code that needs the raw value
+			oo[a].setAttribute('data-sz', numSz);
+			oo[a].textContent = humansize(numSz, false);
```

Comment:
You created a new function called `formatSize`, but here you call `humansize`...?
And you don't call your own function anywhere.